### PR TITLE
Create next version of documentation

### PR DESCRIPTION
**Pull request by paul-vd**

### CODE REVIEW

File: src/theme/BlogPostItem/Footer/index.tsx
Comment (real34) [inline src/theme/BlogPostItem/Footer/index.tsx:1]: **issue:** this isn't related to the new version

**suggestion:** undo this change

**Review by real34 — CHANGES_REQUESTED** [review]
The "Concepts" section should be removed from Remixed version. https://deploy-preview-621--heuristic-almeida-1a1f35.netlify.app/docs/remixed/category/concepts

Once it's done, it's good for me!

File: static/_redirects [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
 # it is automatically generated by the categories
-/docs/*/:category/overview					            /docs/category/:category	                        301
-/docs/*/:category/overview.html					        /docs/category/:category	                        301
+/docs/*/:category/overview					            /docs/category/:category	                        302
+/docs/*/:category/overview.html					        /docs/category/:category	                        302
+
+# Since we introduced documentation versioning, unversioned documentation should redirect to version 2.x
+/docs/*                                             /docs/:splat                                        200!
+/docs/remixed/*                                         /docs/remixed/:splat                                    200!
+/docs/*                                                 /docs/:splat                                        301
```
Comment (paul-vd) [inline static/_redirects:19]: This will ensure that any external links that do not include either `remixed` (still to be discussed on final versioning path) or `2.x` to automatically redirect to `/docs/2.x/*`

File: static/_redirects [hunk 2]
```diff
@@ -1,12 +1,19 @@
...
 # it is automatically generated by the categories
-/docs/*/:category/overview					            /docs/category/:category	                        301
-/docs/*/:category/overview.html					        /docs/category/:category	                        301
+/docs/*/:category/overview					            /docs/category/:category	                        302
+/docs/*/:category/overview.html					        /docs/category/:category	                        302
+
+# Since we introduced documentation versioning, unversioned documentation should redirect to version 2.x
+/docs/2.x/*                                             /docs/2.x/:splat                                    200!
+/docs/remixed/*                                         /docs/remixed/:splat                                200!
+/docs/*                                                 /docs/:splat                                        301
```
Comment (paul-vd) [inline static/_redirects:19]: This will ensure that any external links that do not include either remixed (still to be discussed on final versioning path) or 2.x to automatically redirect to /docs/2.x/*

File: static/_redirects [hunk 3]
```diff
@@ -1,12 +1,19 @@
...
 # it is automatically generated by the categories
-/docs/*/:category/overview					            /docs/category/:category	                        301
-/docs/*/:category/overview.html					        /docs/category/:category	                        301
+/docs/:version/*/:category/overview					            /docs/:version/category/:category	                      302
+/docs/:version/*/:category/overview.html					      /docs/:version/category/:category	                      302
+
+# Since we introduced documentation versioning, unversioned documentation should redirect to version 2.x
+/docs/2.x/*                                             /docs/2.x/:splat                                        200!
+/docs/remixed/*                                         /docs/remixed/:splat                                    200!
+/docs/*                                                 /docs/2.x/:splat                                        302
```
Comment (paul-vd) [inline static/_redirects:19]: This will ensure that any external links that do not include either remixed (still to be discussed on final versioning path) or 2.x to automatically redirect to /docs/2.x/*
